scripts/setup-gcloud-cluster.sh

```diff
@@ -125,9 +130,12 @@ for ip in ${node_ips}; do
   fund_if_empty "${eth_address}" "${environment}"
 done
 
-for ip in ${node_ips}; do
-  wait_for_port "9091" "${ip}"
-done
+# We cannot poll for NAT nodes, because they do not expose 9091 to the outside world
```

Comment:
we can wait for other ports, such as 3001 (health check) or 3000 (REST API)

Reply:
Unfortunately, we cannot. Because 3000 and 3001 will be both up, no matter whether the node has started after funding or not.